code/drasil-gool/lib/GOOL/Drasil/LanguageRenderer/SwiftRenderer.hs

```diff
@@ -1143,4 +1144,4 @@ typeDfltVal Char = litChar ' '
 typeDfltVal String = litString ""
 typeDfltVal (List t) = litList (convType t) []
 typeDfltVal (Array t) = litArray (convType t) []
-typeDfltVal _ = error "Attempt to get default value for type with none."
\ No newline at end of file
+typeDfltVal _ = error "Attempt to get default value for type with none."
```

Comment:
Underrated changes to each files. Drasils codebase has many files that are missing the trailing newline characters.